backend/index.js

```diff
@@ -70,6 +70,116 @@ app.get('/', (req, res) => {
     res.send('game selection');
 });
 
+// devtools
+app.get('/devtools', (req, res) => {
+    res.send('developer tools');
+});
+
+app.get('/words', async (req, res) => {
+    try {
+        const wordsData = await prisma.words.findFirst();
+        if (!wordsData) {
+            wordsData = await prisma.words.create({
+                data: {
+                    usableWords: [],
+                    deletedWords: [],
+                    keywords: [],
+                    classifiers: [],
+                }
+            });
+        } else {
```

Comment:
if you apply the changes to the model that spencer is suggesting, you'll want to modify this endpoint to return many rows instead of only doing a findFirst